lib/govuk_publishing_components/presenters/public_layout_helper.rb

```diff
@@ -69,20 +100,43 @@ def footer_track_actions
         FOOTER_TRACK_ACTIONS
       end
 
-      def generate_data_attribute(link, track_action)
+      def generate_data_attribute(link, track_action, ga4_attributes)
+        if ga4_attributes
+          ga4_link = {
+            "event_name": "navigation",
+            "type": "footer",
+            "index": {
+              "index_link": ga4_attributes[:index_link].to_s,
+              "index_section": ga4_attributes[:index_section].to_s,
+              "index_section_count": "5",
+            }, # //Section.link number notation
```

Comment:
Nitpick: I'm confused by this comment, which line does it refer to?

Reply:
@andysellick Whoops, was meant to remove this - I copied the object straight from the implementation guide then modified it, so those comments are from the implementation guide!